submissions/Temu4/html_css_popup/index.html

```diff
@@ -0,0 +1,301 @@
+                    <div class="app-item__icon">
+                      <img src="img/apps-contact.png" alt="Contact" />
+                    </div>
+                    <div class="app-item__label"><p>Contact</p></div>
+                  </a>
+                </div>
+                <div class="app-item">
+                  <a class="app-item__link" href="#">
+                    <div class="app-item__icon">
+                      <img src="img/apps-massager.png" alt="Massanger" />
+                    </div>
+                    <div class="app-item__label"><p>Massanger</p></div>
+                  </a>
+                </div>
+                <div class="app-item">
+                  <a class="app-item__link" href="#">
+                    <div class="app-item__icon">
+                      <img src="img/apps-keep.png" alt="Keep" />
+                    </div>
+                    <div class="app-item__label"><p>Keep</p></div>
+                  </a>
+                </div>
+                <div class="app-item">
+                  <a class="app-item__link" href="#">
+                    <div class="app-item__icon">
+                      <img src="img/apps-jumboard.png" alt="Jambord" />
+                    </div>
+                    <div class="app-item__label"><p>Jambord</p></div>
+                  </a>
+                </div>
+                <div class="app-item">
+                  <a class="app-item__link" href="#">
+                    <div class="app-item__icon">
+                      <img src="img/apps-classroom.png" alt="Classroom" />
+                    </div>
+                    <div class="app-item__label"><p>Classroom</p></div>
+                  </a>
+                </div>
+                <div class="app-item">
+                  <a class="app-item__link" href="#">
+                    <div class="app-item__icon">
+                      <img src="img/apps-earth.png" alt="Planet" />
+                    </div>
+                    <div class="app-item__label"><p>Planet</p></div>
+                  </a>
+                </div>
+                <div class="app-item">
+                  <a class="app-item__link" href="#">
+                    <div class="app-item__icon">
+                      <img src="img/apps-collection.png " alt="Collections" />
+                    </div>
+                    <div class="app-item__label"><p>Collections</p></div>
+                  </a>
+                </div>
+                <div class="apps-box__even-more even-more">
+                  <a class="even-more__link" href="#">Even more from Booble</a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </li>
+        <!-- Link for messages -->
+        <li class="header__link link">
+          <a
+            href="#"
+            title="Уведомлений Booble: 0"
+            class="link__label"
+            tabindex="0"
+            ><span class="link__content" tabindex="-1"
```

Comment:
Do not transfer each attribute to a new line.
When the focus is on the element the tag is highlighted but not the icon.

Reply:
It's config of my IDE. For me it's useful.

Reply:
It's easier to read. I do in the same way.